src/routes.ts

```diff
@@ -33,6 +33,7 @@ interface BaseRequestAPICall extends BaseAPICall {
   headers?: Headers
   proxy?: any, // TODO: use interface not any
   download?: boolean
+  justCookies?: boolean
```

Comment:
Maybe `returnOnlyCookies`? Not sure if this is the best variable name

Reply:
fixed